src/translation/Translator.as

```diff
@@ -189,6 +189,7 @@ public class Translator {
 			if (mode == 'val') val += extractQuotedString(line);
 		}
 		if (mode == 'val') dict[key] = val; // recordPairIn(key, val, dict);
+		delete dict['']; // remove the empty-string metadata entry, if present.
```

Comment:
I could have prevented the entry from ever making it into the dictionary, but I would have had to put the key check in both places -- there's one inside the loop as well. this seemed more readable and arguably more efficient. But mostly more readable :)